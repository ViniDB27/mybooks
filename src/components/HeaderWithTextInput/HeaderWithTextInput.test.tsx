import render from "react-test-renderer";

import { HeaderWithTextInput } from "./index";

describe("<HeaderWithTextInput />", () => {
  const tree = render
    .create(<HeaderWithTextInput value="Mock" onChangeText={() => {}} />)
    .toJSON();

  it("renders correctly", () => {
    expect(tree).toMatchSnapshot();
  });
});
