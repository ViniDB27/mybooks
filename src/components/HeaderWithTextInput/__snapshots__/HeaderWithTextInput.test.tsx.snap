// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderWithTextInput /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0Af",
      "height": 70,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Busque por um livro"
    rejectResponderTermination={true}
    style={
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 5,
        "height": 40,
        "paddingLeft": 10,
        "width": "80%",
      }
    }
    underlineColorAndroid="transparent"
    value="Mock"
  />
</View>
`;
